CRM/Civicase/Helper/CaseCategory.php

```diff
@@ -47,13 +50,13 @@ public static function getCategoryName($caseId) {
   public static function getCategoryNameForCaseType($caseTypeId) {
     $caseTypeCategories = CaseType::buildOptions('case_type_category');
 
-    $result = civicrm_api3('CaseType', 'getsingle', [
+    $result = civicrm_api3('CaseType', 'getvalue', [
```

Comment:
I think even getvalue throw an exception if no result found, so I think probably better to add Try, catch block